src/rgw/rgw_admin.cc

```diff
@@ -5485,7 +5523,13 @@ int main(int argc, const char **argv)
 
         for (vector<rgw_bucket_dir_entry>::iterator iter = result.begin(); iter != result.end(); ++iter) {
           rgw_bucket_dir_entry& entry = *iter;
+	  formatter->open_object_section("entry");
           encode_json("entry", entry, formatter);
+	  if (extra_info) {
+	    rgw_obj obj(bucket_info.bucket, entry.key);
+	    show_obj_manifest(store, bucket_info, obj, formatter);
```

Comment:
conflict with Dan's zipper work here, try passing `store->getRados()` instead:
> /home/jenkins-build/build/workspace/ceph-pull-requests/src/rgw/rgw_admin.cc:1134:29: error: no matching function for call to 'RGWObjectCtx::RGWObjectCtx(RGWRados*&)'
   RGWObjectCtx obj_ctx(store);
                             ^
In file included from /home/jenkins-build/build/workspace/ceph-pull-requests/src/rgw/rgw_admin.cc:38:
/home/jenkins-build/build/workspace/ceph-pull-requests/src/rgw/rgw_rados.h:213:12: note: candidate: 'RGWObjectCtx::RGWObjectCtx(rgw::sal::RGWRadosStore*, void*)'
   explicit RGWObjectCtx(rgw::sal::RGWRadosStore *_store, void *_s) : store(_store), s(_s) {}